src/core/commands/handle_interaction.rs

```diff
@@ -108,6 +111,9 @@ pub async fn handle_command(ctx: Arc<Context>, mut command: ApplicationCommand)
                 .await
                 .map(|_| ProcessResult::Success)
         }
+        //TODO: bucket baby
```

Comment:
Try to resolve this one, either create and use a bucket or argue why it's not necessary.